src/server/services/brp.ts

```diff
@@ -80,6 +66,11 @@ export function transformBRPNotifications(data: BRPData, compareDate: Date) {
     }
   );
 
+  const documentsExpiringDuringPeak = data.identiteitsbewijzen?.filter(
+    (document) => {
+      return new Date(document.datumAfloop) < new Date('2024-10-01') 
```

Comment:
Kun je deze in een constant zetten? Zie ook https://github.com/Amsterdam/mijn-amsterdam-frontend/pull/1108/files/93ac3fb0642bb9d30fccdadf960ca511d86fc873#r1435041810

Reply:
Heh lijkt alsof jij naar oud doc kijkt. Ik heb dat gedaan.  
 
```
const documentsExpiringDuringPeak = data.identiteits
bewijzen?.filter(
    (document) => {
      const afloop = new Date(document.datumAfloop)
      return ID_BEWIJS_PEAK_DATE_START <afloop && new Date(document.datumAfloop) < ID_BEWIJS_PEAK_DATE_END 
    }
  );
```